x/group/server/invariants.go

```diff
@@ -0,0 +1,94 @@
+package server
+
+import (
+	"fmt"
+	"math"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+
+	"github.com/regen-network/regen-ledger/orm"
+	"github.com/regen-network/regen-ledger/types"
+	"github.com/regen-network/regen-ledger/x/group"
+)
+
+func (s serverImpl) RegisterInvariantsHandler(ir sdk.InvariantRegistry) {
+	ir.RegisterRoute(group.ModuleName, "Tally-Votes", s.TallyVotesInvariant())
+}
+
+func (s serverImpl) AllInvariants() sdk.Invariant {
+	return func(ctx sdk.Context) (string, bool) {
+		return s.TallyVotesInvariant()(ctx)
+	}
+}
+
+func (s serverImpl) TallyVotesInvariant() sdk.Invariant {
+	return func(sdkCtx sdk.Context) (string, bool) {
+		var msg string
+		var broken bool
+		ctx := types.Context{Context: sdkCtx}
+		if ctx.BlockHeight()-1 < 0 {
+			return sdk.FormatInvariant(group.ModuleName, "Tally-Votes", "Not enough blocks to perform TallyVotesInvariant"), false
+		}
+		sdkCtx = sdkCtx.WithBlockHeight(ctx.BlockHeight() - 1)
+		it1, err := s.proposalTable.PrefixScan(ctx, 1, math.MaxUint64)
```

Comment:
should we rename it to be more consistent with the proposals var naming (ie `curProposals`)?
```suggestion
		curIt, err := s.proposalTable.PrefixScan(ctx, 1, math.MaxUint64)
```